mopidy_musicbox_webclient/static/js/functionsvars.js

```diff
@@ -343,12 +343,9 @@ function resultsToTables(results, target, uri) {
                     }
                     newalbum = [];
                     newtlids = [];
-                    if (results[i].album) {
-                        coversList.push([results[i].album, i]);
-                    }
                 } //newalbum length
                 if (results[i].album) {
-                    coversList.push([results[i].album, i]);
+                    coversList.push([results[i].uri, i]);
```

Comment:
I couldn't see why it was necessary to have the same statement inside the ``if``, and then again immediately outside it. Did I miss something there?